source/agora/flash/UpdateSigner.d

```diff
@@ -480,17 +483,19 @@ public class UpdateSigner
         // Note that an update tx with seq 0 do not exist.
         if (this.seq_id == 0)
         {
-            return sign(this.kp.secret, this.conf.pair_pk, nonce_pair_pk,
-                priv_nonce.update.v, update_tx.getChallenge());
+            const sig_hash = cast(SigHash)(SigHash.Single | SigHash.AnyoneCanPay);
```

Comment:
Nope, a flag is not an enum. I wish D had something like C#'s flag, but it currently doesn't.
Casting an out-of-range value to an `enum` type is like taking a chainsaw to the type system, and I'd rather we do not do that.

Reply:
I explicitly defined them.